.prow.yaml

```diff
@@ -1029,7 +1029,7 @@ presubmits:
         - name: KUBERMATIC_EDITION
           value: ee
         - name: VERSIONS_TO_TEST
-          value: "v1.22.1"
+          value: "v1.21.0"
```

Comment:
Let's use the latest patch release -- v1.21.4.

Reply:
The latest 1.21 patch release supported by KKP in creation seems to be 1.21.3: https://github.com/kubermatic/kubermatic/blob/e56b93784e25f9eaa6c820e4a8f95fd235a8efaf/pkg/controller/operator/common/defaults.go#L207-L210

Reply:
My bad. 1.21.3 is okay as well.